core/src/main/java/feign/template/QueryTemplate.java

```diff
@@ -119,7 +120,7 @@ private QueryTemplate(
       Charset charset,
       CollectionFormat collectionFormat) {
     super(template, false, true, true, charset);
-    this.name = name;
+    this.name = new Template(name, false, true, false, charset);
```

Comment:
At some point we should replace this booleans with enums that communicate what is going on..
`new Template(name, Resolution.REQUIRED, Encoding.REQUIRED, Encoding.DISABLED, charset);`

Not a problem for today

Reply:
Why wait?  I’ll add it.

Reply:
Updated